app/assets/stylesheets/views/_colhub.scss

```diff
@@ -0,0 +1,9 @@
+.colhub__announcement {
+  background-color: govuk-colour("light-grey");
```

Comment:
Do we have any other examples of using this background colour with the standard link styles? I notice that the blue against the grey fails AAA (passes AA). It's not a WCAG fail but something to look at later, maybe we should follow the footer link styles for this announcement banner.  The footer uses this grey background colour but the links there are black.  (Just for completeness, the focus style yellow fails the colour contrast check here the against the grey background although it's also not a WCAG fail since the underline is still there to indicate focus.)

Reply:
Is AA compliance acceptable for our link colours or do they need to be AAA?

Reply:
AA is acceptable but it might be advisable to revise the link colours later to follow something similar to what the footer does. But this requirement has now been captured [here](https://trello.com/c/MlXyrLwz/746-remove-fix-inline-changes-for-cost-of-living-hub-page-s) so resolving this comment as well.